include/riak_kv_ts.hrl

```diff
@@ -29,12 +29,12 @@
 %% For dialyzer types
 -include_lib("riak_ql/include/riak_ql_ddl.hrl").
 
--define(SQL_SELECT, #riak_select_v1).
--define(SQL_SELECT_RECORD_NAME, riak_select_v1).
+-define(SQL_SELECT, #riak_select_v2).
```

Comment:
I'm assuming all of this versioning will be addressed in the upgrade/downgrade PR? 😉